### ISSUE TITLE
Evaluate @ and + commands to remove from vRWOT

### ISSUE THREAD

**Issue by ChristopherA** [WebOfTrustInfo/prototype_vRWOT#16]
There are far too many OOC @ commands that are available to all users that are not applicable to vRWOT. There are also a number that should even be removed from base SkotOS and TheGables.

This task is to list and evaluate all the @ command to:

* for removal from vRWOT
* for removal from base SkotOS
* for adding to TheGables
* which ones are easy to remove as they are only verb objects (no associated prop-containers or intertwined commands).

Once we have this list, discuss with @ChristopherA and @noahgibbs on the priority of actually move these to plugin(s). Clearly we don't want one plugin per occ @ verb — we'll want to group related ones together.

We should also at some point do same for the + commands, but are a lower priority as few people see them.

Related but separate, we should identify and evaluate any user available commands that are not needed for vRWOT or base Skotos, in particular those that are not implemented in Merry and are hard coded. These will require assistance from @noahgibbs to remove so that they can be replaced with Merry based versions.

This is a good ongoing project for @saraht45.

**Comment by saraht45** [comment]
I have started compiling @commands [here](https://docs.google.com/spreadsheets/d/1xmug-AwhrhjnOcLQGXb6lMe1ZieYz2f4Z51C2ytKegI/edit?usp=sharing) and documenting relevant info.

I haven't added hard coded commands to the list yet (@page, @help).

**Comment by ChristopherA** [comment]
I forgot about `@remember` & `@forget` 😀

Those are real candidates for removal from vRWOT but will be harder as I don’t think they are in Merry. 

And for longer term game usage I’d really like them not only be available from Merry but also function for future game stages where we can give a body character memories, not the user. So questionable how much we want to preserve the legacy.

**Comment by ChristopherA** [comment]
A question for @jandrieu — we have a bunch of different across-the-server OOC chat channels for games, and even more chat channels for admins. Think IRC but inside the mud — no objects. Should we remove them from regular users?

**Comment by saraht45** [comment]
It wouldn't be difficult to code a memories system in Merry and have it operate the exact same way.

**Comment by jandrieu** [comment]
@ChristopherA Hmmm... I think they could be useful. Is there some technical advantage of removing them?

**Comment by ChristopherA** [comment]
There is utility of keeping chat lines for moderators to make announcements or back channels that guests don’t see — we’d probably not remove those. 

But chat lines becomes yet another confusing text interface inside a text interface for regular users. Fine for gamers who spend 10+ hours a week in the game to have a OOC back channels, but our guests will not be “in-character” and will be unlikely to be spending hundreds of hours playing.

**Comment by noahgibbs** [comment]
Yeah, the old Marrach memories system looks pretty incongruous at this point - something that should clearly be done in Merry that's still in the very lowest DGD levels. We could remove it fairly easily, looks like.

**Comment by ChristopherA** [comment]
I love the memories feature, but I don’t think we need to try to preserve the legacy code and turn it into plugin. I do think it is command baggage that vRWOT users don’t need.

Thus I suggest we document its functionality and kill it from base SkotOS, TheGables and vRWOT. I can commission @saraht45 for a new plugin to reimplement this feature using Merry in the future when ChatTheatre can afford to implement Stages.

**Comment by jandrieu** [comment]
I like moving to Merry. That feels like a good refactoring.

At this point, I'd focus on getting the plumbing right: that the right functionality is provided at the right layer. I'd avoid worrying about the UX concerns as we have a separate workflow to figure out what we expose to the user in the interface. Shifting to Merry seems like a good adjustment along those lines.

**Comment by saraht45** [comment]
Here are my suggestions for how to handle the current @commands. I'll create a separate list for hardcoded dgd commands that can be removed and replaced with Merry commands.

**Keep**
@assist, @bug, @idea, @social, @typo, @gc, @allow, @always, @consent, @deny, @never, @elude, @adverb, @verb, @cont, @a, @b, @c, @d, @e, @no, @yes, @afk, @away, @busy, @wb

**Remove for all**
@addmemo, @calendar, @sk, @melee, @chargen, @start, @release, @goto, @status, @supereyes, @gamevote, @themeedit, @themetest, @uptime, @sort, @mcombat, @keepalive (replace with @toggle)

**Keep but remove for vRWOT**
@fadetoblack, @ooc, @oocprefix, @dress, @undress

**Keep for all but needs work**
@toggle, @forget, @recall, @remember, @time, @compose, @readall
All read/write commands need rewritten since they are dependent on bilbo and we don't want bilbo.

**Move to plugin**
@article, @catalog, @speak
